### PR TITLE
[docs] Correct prerequisites and cqlsh connection command

### LINKED ISSUE #3840: [Docs] Fix TLS "Connect to cluster" documentation
There are some mistakes in the "Connect to cluster" page for TLS: https://docs.yugabyte.com/latest/secure/tls-encryption/connect-to-cluster/

1. In prerequisites, generating yugabytedb.crt and yugabytedb.key links to "creating server certificates". Instead, it should link to "creating client certificates".

2. These prerequisites are only for YSQL. For YCQL, just the server certificates need to be generated.

3. cqlsh instructions for remote clusters: port should be 9042 and not 5433. Also, command is incorrect - cqlsh does not use -h, -p and -U options for host, port and username. It should just be `./bin/cqlsh 127.0.0.2 9042 --ssl`

### PR DESCRIPTION
**Pull request by stevebang**
- Fix links to client certificate files
- Edit and correct prerequisites
- Correct cqlsh connection command

Fixes #3840

### CODE REVIEW

File: docs/content/latest/secure/tls-encryption/connect-to-cluster.md [hunk 1]
```diff
@@ -19,16 +19,18 @@ To connect CLIs, tools, and APIs to a remote YugabyteDB cluster when client-to-s
 
 ## Prerequisites
 
-Before you can enable and use server-to-server encryption, you need to create and configure server certificates for each node of your YugabyteDB cluster. For information, see [Create client certificates](../client-certificates).
+Before you can use client certificates to connect to your YugabyteDB clusters, server-to-server encryption must be enabled. For details, see [Enable server-to-server encryption](../server-to-server).
```
Comment (ndeodhar) [inline docs/content/latest/secure/tls-encryption/connect-to-cluster.md]: server-to-server and client-to-server encryption must be enabled.

File: docs/content/latest/secure/tls-encryption/connect-to-cluster.md [hunk 2]
```diff
@@ -19,16 +19,18 @@ To connect CLIs, tools, and APIs to a remote YugabyteDB cluster when client-to-s
...
 
-For each client that will connect to a YugabyteDB cluster, you need the following three files to be accessible on the client computer.
+Also, you need to create the required client certificates. For details, see [Create client certificates](../client-certificates).
 
-- `root.crt` — root certificate file
-  - To generate, see [Prepare root configuration file](../prepare-nodes/#generate-root-config)
+Each client that connects to a YugabyteDB cluster needs the following three files to be accessible on the client computer.
+
+- `ca.crt` — root certificate file
```
Comment (ndeodhar) [inline docs/content/latest/secure/tls-encryption/connect-to-cluster.md]: Is this correct for YSQL? Or does YSQL filename need to be root.crt?

File: docs/content/latest/secure/tls-encryption/connect-to-cluster.md [hunk 3]
```diff
@@ -19,16 +19,18 @@ To connect CLIs, tools, and APIs to a remote YugabyteDB cluster when client-to-s
...
+  - To generate, see [Generate the root certificate file](../server-certificates/#generate-the-root-certificate-file)
 - `yugabytedb.crt` — private node certificate
-  - To generate, see [Generate private key for each node](../prepare-nodes/#generate-private-key-for-each-node)
+  - To generate, see [Generate client private key and certificate](../client-certificates/#generate-private-key-and-certificate)
```
Comment (ndeodhar) [inline docs/content/latest/secure/tls-encryption/connect-to-cluster.md]: These two are only needed if you are going to use YSQL. We should call that out explicitly.

Alternatively, we can change this to have different tabs for YCQL and YSQL and include instructions separately. In my opinion, that might be easier to understand.

File: docs/content/latest/secure/tls-encryption/connect-to-cluster.md (hunk 1 above)
Comment (stevebang) [reply docs/content/latest/secure/tls-encryption/connect-to-cluster.md]: Good catch. fixed

File: docs/content/latest/secure/tls-encryption/connect-to-cluster.md (hunk 3 above)
Comment (stevebang) [reply docs/content/latest/secure/tls-encryption/connect-to-cluster.md]: I just add "(YSQL and YCQL)" and "(YSQL only)" after the file names. I think that makes it clear.

File: docs/content/latest/secure/tls-encryption/connect-to-cluster.md (hunk 2 above)
Comment (stevebang) [reply docs/content/latest/secure/tls-encryption/connect-to-cluster.md]: Good question -- `root.crt` was my idea, but the code is looking for `ca.crt`.  I did a search and found two more instances with the wrong file name.

File: docs/content/latest/secure/tls-encryption/connect-to-cluster.md [hunk 4]
```diff
@@ -19,16 +19,18 @@ To connect CLIs, tools, and APIs to a remote YugabyteDB cluster when client-to-s
...
 
-- `root.crt` — root certificate file
-  - To generate, see [Prepare root configuration file](../prepare-nodes/#generate-root-config)
-- `yugabytedb.crt` — private node certificate
-  - To generate, see [Generate private key for each node](../prepare-nodes/#generate-private-key-for-each-node)
-- `yugabytedb.key` — private node key
-  - To generate, see [Generate private key for each node](../prepare-nodes/#generate-private-key-for-each-node)
+Each client that connects to a YugabyteDB cluster needs the following files to be accessible on the client computer.
+
+- `ca.crt` — root certificate file (YSQL and YCQL)
```
Comment (ndeodhar) [inline docs/content/latest/secure/tls-encryption/connect-to-cluster.md]: Did you confirm that naming the file as `ca.crt` works for YSQL?

Also, I still find this a bit hard to read (with YCQL and YSQL mentioned for every step), but it's okay to handle it later.

**Pull request merged**